### PR TITLE
feat(specs): add /schedule endpoint

### PR DESCRIPTION
**Pull request by febeck**
## 🧭 What and Why

Add documentation for newly added `/schedule` endpoint

🎟 JIRA Ticket: [OPTIM-1160]

### Changes included:

- Add new endpoint to the list
- Add new parameter `scheduledAt`
- Add new response payload

## 🧪 Test

Added new test file with sample

[OPTIM-1160]: https://algolia.atlassian.net/browse/OPTIM-1160?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by shortcuts** [comment]
Hey, for that new endpoint, you need to add a test file (see [docs](https://api-clients-automation.netlify.app/docs/contributing/testing/common-test-suite#requests-tests)) here https://github.com/algolia/api-clients-automation/tree/main/tests/CTS/requests/abtesting with the name of the operation

File: specs/abtesting/common/parameters.yml [hunk 1]
```diff
@@ -33,6 +38,11 @@ updatedAt:
   description: Date and time when the A/B test was last updated, in RFC 3339 format.
   example: 2023-06-15T15:06:44.400601Z
 
+scheduledAt:
+  type: string
+  description: Date and time when the A/B test scheduled to start, in RFC 3339 format.
```
Comment (millotp) [inline specs/abtesting/common/parameters.yml:43]:
```suggestion
  description: Date and time when the A/B test is scheduled to start, in RFC 3339 format.
```
?

File: specs/abtesting/paths/scheduleABTest.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - editSettings
+  summary: Schedule an A/B test
+  description: |
+    Schedule an A/B test to be started at a later time.
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          title: addABTestsRequest
```
Comment (millotp) [inline specs/abtesting/paths/scheduleABTest.yml:15]: could you rename this type so it doesn't clash with the one in `abtests.yml` pls ?

**Review by millotp — COMMENTED** [review]
this looks really good thanks for the contribution !

File: specs/abtesting/paths/scheduleABTest.yml (hunk 2 above)
Comment (febeck) [reply specs/abtesting/paths/scheduleABTest.yml:15]: Nice catch, thanks for that!

File: specs/abtesting/common/parameters.yml (hunk 1 above)
Comment (febeck) [reply specs/abtesting/common/parameters.yml:43]: Oups, fixed. Thanks

**Review by JonathanMontane — DISMISSED** [review]

**Comment by millotp** [comment]
Hey @febeck, is this ready to be merged or is the API not ready yet ?

**Comment by febeck** [comment]
> Hey @febeck, is this ready to be merged or is the API not ready yet ?

Hey @millotp this is not yet to be merged as there is still some work to be done around this endpoint. It will probably stay here in draft for still a couple of weeks at least.

Don't worry about rebasing or checking on this. I'll put at is as "ready for review" when it's actually ready to be merged. 

I am sorry if this has disturbed your workflow, I should have noted it when opening the PR

**Comment by febeck** [comment]
@millotp  and @shortcuts this is finally ready to be merged.

I have CI failing but I don't really get what the issue is. Could one of you help me with that, please ?

File: specs/abtesting/common/schemas/ScheduledABTestResponse.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+ScheduledABTestResponse:
```
Comment (millotp) [inline specs/abtesting/common/schemas/ScheduledABTestResponse.yml:1]: should this be called `ScheduleABTestResponse` to match the operationId ?

**Review by millotp — COMMENTED** [review]
small question, looks good otherwise !

File: specs/abtesting/common/schemas/ScheduledABTestResponse.yml (hunk 3 above)
Comment (febeck) [reply specs/abtesting/common/schemas/ScheduledABTestResponse.yml:1]: Good call, I am fixing this and pushing the update

**Review by millotp — APPROVED** [review]
perfect !

**Comment by shortcuts** [comment]
Hey, this feature will be available in the next minor of the API clients, later today :)

**Pull request merged**